### PR TITLE
Remove reactNativeCliInstall step

### PR DESCRIPTION
**Pull request by sfernandez11**
## Summary
* Removed global dependency of `react-native-cli` ([Update cli doc](https://github.com/react-native-community/cli#updating-the-cli))
* Replace react-native command with `npx react-native ...` (RN community indication)
* All tested and working
* Update again some Firebase configuration 
* Updated CHANGELOG for 1.4.0 version
* Fixed script in Generator and Bootstraped Project
* Updated `androidx.appcompat` dependency in Android project

## Related Issues
* [Issue](https://github.com/facebook/react-native/issues/29976) with `:CFBundleIdentifier` in iOS project due to use global `react-native-cli` package instead of the local one.
* Issue with Images in Xcode 12 and iOS 14 (solved in RN 0.63 version)

## Screenshots
* No visual changes

## Trello Card
https://trello.com/c/kz8znNx5/199-eliminar-dependencia-global-de-react-native-cli

### CODE REVIEW

File: generators/app/tasks/appSetup/featuresFiles/firebasePerformanceSetup.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 module.exports = function firebasePerformanceSetup() {
   let buildGradleContent = this.fs.read(`${this.projectName}/android/build.gradle`);
   buildGradleContent = buildGradleContent.replace(
-    'dependencies {',
-    "dependencies {\n\t\tclasspath 'com.google.firebase:perf-plugin:1.3.1'"
+    '// NOTE: Do not place your application dependencies here; they belong',
+    "classpath 'com.google.firebase:perf-plugin:1.3.1'\n\t\t// NOTE: Do not place your application dependencies here; they belong"
```
Comment (mcavo) [inline generators/app/tasks/appSetup/featuresFiles/firebasePerformanceSetup.js:5]: the comment is duplicated and it seems incomplete, if im not wrong 😅

**Review by mcavo — COMMENTED** [review]
lgtm just one comment 😄

File: generators/app/tasks/appSetup/featuresFiles/firebasePerformanceSetup.js (hunk 1 above)
Comment (sfernandez11) [reply generators/app/tasks/appSetup/featuresFiles/firebasePerformanceSetup.js:5]: The idea of this replace is to transform the string: `// NOTE: Do not place your application dependencies here; they belong` into `classpath 'com.google.firebase:perf-plugin:1.3.1'\n\t\t// NOTE: Do not place your application dependencies here; they belong` for adding the `firebase-performance` dependency
Before
```
...
// NOTE: Do not place your application dependencies here; they belong
...
```
After
```
...
classpath 'com.google.firebase:perf-plugin:1.3.1'
// NOTE: Do not place your application dependencies here; they belong
...
```

**Review by mcavo — DISMISSED** [review]
🥇

**Pull request merged**